aten/src/ATen/native/quantized/TensorFactories.cpp

```diff
@@ -59,7 +59,7 @@ Tensor empty_per_channel_affine_quantized(
       options.has_dtype(),
       "Must provide data type for Tensor creation functions.");
   QuantizerPtr quantizer = make_per_channel_affine_quantizer(
-          scales, zero_points, axis, typeMetaToScalarType(options.dtype()));
+          scales.to(options.device()), zero_points.to(options.device()), axis, typeMetaToScalarType(options.dtype()));
```

Comment:
btw, is this the only place we need this?

Reply:
its only needed in per_channel and everything I'm seeing flows through this specific constructor